### ISSUE TITLE
"a JS error occurred in the main process"

### ISSUE THREAD

**Issue by serjanbiogenic** [notion-enhancer/notion-enhancer#474]
**describe the bug**
After installing notion-enhancer i get a JS error occurred in the main process

Uncaught Exception:
Error: Cannot find module '/usr/local/lib/node_modules/notion-enhancer/pkg/loader.js'
Require stack:
- /Applications/Notion.app/Contents/Resources/app/config.js
- /Applications/Notion.app/Contents/Resources/app/main/crashReporter.js
- /Applications/Notion.app/Contents/Resources/app/main/main.js
- 
at Module._resolveFilename (internal/modules/cjs/loader.js:972:15)
at Function.n._resolveFilename (electron/js2c/browser_init.js:249:921)
at Module._load (internal/modules/cjs/loader.js:848:27)
at Function.f._load (electron/js2c/asar_bundle.js:5:12694)
at Module.require (internal/modules/cjs/loader.js:1032:19)
at require (internal/modules/cjs/helpers.js:72:18)
at Object.<anonymous> (/Applications/Notion.app/Contents/Resources/app/config.js:8:1)
at Module._compile (internal/modules/cjs/loader.js:1152:30)
at Object.Module._extensions..js (internal/modules/cjs/loader.js:1173:10)
at Module.load (internal/modules/cjs/loader.js:992:32)

**platform**
macOS 11.2.3

**Comment by dragonwocky** [comment]
sounds like you missed something in the installation or some files went missing after installation. please provide screenshots of your attempts to install the enhancer.

**Comment by ShigaHaruki** [comment]
![RTMFC8ZXLO ($HW9J)E7XAF](https://user-images.githubusercontent.com/68691294/116504919-1996e600-a8ec-11eb-89fa-089a66e75b76.jpg)

**Comment by ShigaHaruki** [comment]
> sounds like you missed something in the installation or some files went missing after installation. please provide screenshots of your attempts to install the enhancer.

我安装之后出现了这种情况
![RTMFC8ZXLO ($HW9J)E7XAF](https://user-images.githubusercontent.com/68691294/116504977-3fbc8600-a8ec-11eb-8ba3-eaba15016a5d.jpg)
我也进行了修改 ： https://www.notion.so/hack-notion-enhancer-v0-10-2-to-work-with-notion-s-v2-0-13-app-a4ab1267e67b4126b6448641ca4a0041

**Comment by dragonwocky** [comment]
@shigaharuki this is a screenshot of you opening notion. please show screenshots of the files you edited and the commands you ran.

**Comment by emilianovb** [comment]
@dragonwocky I'm using the latest versions of Notion, node.js and the enhancer. I have completely uninstalled and reinstalled several times. The only commands I used were the terminal commands in the installation page, and 
`3. replace line 108 of `notion-enhancer/mods/core/client.js` with
`!document.querySelector('.notion-topbar > div[style*="display: flex"]')`
4. add the following to line 19 of `notion-enhancer/mods/core/createWindow.js`:

    ```jsx
    __exports.getIndexUrl = require(`${helpers
        .getNotionResources()
        .replace(/\\/g, '/')}/app/helpers/urlHelpers.js`).getIndexUrl;
    ````
From [https://www.notion.so/hack-notion-enhancer-v0-10-2-to-work-with-notion-s-v2-0-13-app-a4ab1267e67b4126b6448641ca4a0041](url)

**Comment by dragonwocky** [comment]
@emilianovb just to check.... did you do steps 3 and 4 by actually editing a file or did you just run them in terminal?

**Comment by emilianovb** [comment]
@dragonwocky I edited the files. The enhancer worked flawlessly before the windows update, though I'm not sure that the update is the cause of this.

**Comment by emilianovb** [comment]
Update- I solved the issue by doing this:

1. Deleted the following folders; 
      - Notion, npm, npm-cache in C:\Users\x\AppData\Roaming
      - all notion files in C:\Users\x\AppData\Local
2. Force 'search for updates' and updated Windows 10 to 21H1
3. Installed Node.js v16 (not the recommended LTS)
3. Uninstall and reinstall the notion app
4. Ran `notion-enhancer apply -y` in the terminal

That got rid of the JS error, then I followed [this](https://www.notion.so/hack-notion-enhancer-v0-10-2-to-work-with-notion-s-v2-0-13-app-a4ab1267e67b4126b6448641ca4a0041) to open the app.
